src/app.ts

```diff
@@ -89,6 +99,58 @@ export class App {
 
       oidcConfig.userManagerSettings.authority = openIdConnectRoute;
     }
+
+    const latestSolutionUri: string = window.sessionStorage.getItem('latestSolutionUri');
+    const latestActiveDiagramName: string = window.sessionStorage.getItem('latestActiveDiagramName');
+
+    if (latestSolutionUri && latestActiveDiagramName) {
```

Comment:
Extract Business Rule
```suggestion
    const latestSolutionUriExists: boolean = latestSolutionUri !== null && latestSolutionUri !== undefined;
    const latestActiveDiagramName: boolean = latestActiveDiagramName !== null && latestActiveDiagramName !== undefined;
    if (latestSolutionUriExists && latestActiveDiagramNameExists) {
```